### PR TITLE
Embed alphabet in model file

### PR DESCRIPTION
**Pull request by reuben**
Let's see how the tests go.

### CODE REVIEW

**Comment by reuben** [comment]
I'm getting empty transcripts, and sometimes segfaults in the TFLite tests. Yet it always works locally and valgrind is clean... D:

**Comment by reuben** [comment]
Can't reproduce locally on a Pi 4 either :(

**Comment by reuben** [comment]
Of course once I add some diagnostic prints, it passes all tests.

**Comment by lissyx** [comment]
I'll give it a try locally with valgrind and/or rr chaos mode

**Comment by reuben** [comment]
All green, for the second time now.

File: DeepSpeech.py [hunk 1]
```diff
@@ -780,12 +780,11 @@ def export():
     graph_version = int(file_relative_read('GRAPH_VERSION').strip())
     assert graph_version > 0
 
-    # Reshape with dimension [1] required to avoid this error:
-    # ERROR: Input array not provided for operation 'reshape'.
     outputs['metadata_version'] = tf.constant([graph_version], name='metadata_version')
     outputs['metadata_sample_rate'] = tf.constant([FLAGS.audio_sample_rate], name='metadata_sample_rate')
     outputs['metadata_feature_win_len'] = tf.constant([FLAGS.feature_win_len], name='metadata_feature_win_len')
     outputs['metadata_feature_win_step'] = tf.constant([FLAGS.feature_win_step], name='metadata_feature_win_step')
+    outputs['metadata_alphabet'] = tf.constant([Config.alphabet.serialize()], name='metadata_alphabet')
```
Comment (lissyx) [inline DeepSpeech.py]: :100:

File: native_client/ctcdecode/__init__.py [hunk 2]
```diff
@@ -18,9 +18,18 @@ class Scorer(swigwrapper.Scorer):
...
-        err = self.init(alpha, beta, model_path, trie_path, alphabet.config_file())
+        serialized = alphabet.serialize()
+        native_alphabet = swigwrapper.Alphabet()
+        err = native_alphabet.deserialize(serialized, len(serialized))
         if err != 0:
-          raise ValueError("Scorer initialization failed with error code {}".format(err), err)
+            raise ValueError("Error when deserializing alphabet.")
+
+        err = self.init(alpha, beta,
+                        model_path.encode('utf-8'),
```
Comment (lissyx) [inline native_client/ctcdecode/__init__.py]: nit: might do funny stuff on windows :)

File: taskcluster/examples-base.tyml [hunk 3]
```diff
@@ -30,7 +30,7 @@ then:
     image: ${build.docker_image}
 
     env:
-      DEEPSPEECH_MODEL: "https://github.com/lissyx/DeepSpeech/releases/download/test-model-0.6.0a10/models.tar.gz"
+      DEEPSPEECH_MODEL: "https://github.com/reuben/DeepSpeech/releases/download/v0.6.0-alpha.11/models.tar.gz"
```
Comment (lissyx) [inline taskcluster/examples-base.tyml]: nit: technically this will be for alpha 12 :)

**Pull request merged**